test/collections/DequeParity.compopts

```diff
@@ -0,0 +1,2 @@
+-sisBounded --main-module=DequeParity
+--main-module=DequeParity
```

Comment:
Good question, that was from before, but I do need it to run both with and withou t`-sisBounded` set, and I wasn't sure if blank lines get counted.